src/script/hydra-eval-jobset

```diff
@@ -709,36 +744,20 @@ sub checkJobsetWrapped {
         return;
     }
 
-    die "Jobset contains a job with an empty name. Make sure the jobset evaluates to an attrset of jobs.\n"
```

Comment:
Is this a relevant check? Kind of surprising to see it gone

Reply:
@Ma27 Do you know about this? Maybe this is something that should be restored in the new streaming version? Or maybe its fine?

Reply:
If I had to guess, this probably just got removed since `$jobs` got replaced by an iterator?
But yeah, seems reasonalbe to add it back.